paddlenlp/transformers/model_utils.py

```diff
@@ -54,24 +68,40 @@ def register_base_model(cls):
 @six.add_metaclass(InitTrackerMeta)
 class PretrainedModel(Layer, GenerationMixin):
     """
-    The base class for all pretrained models. It provides some attributes and
-    common methods for all pretrained models, including attributes `init_config`,
-    `config` for initialized arguments and methods for saving, loading.
-    It also includes some class attributes (should be set by derived classes):
-    - `model_config_file` (str): represents the file name for saving and loading
-      model configuration, it's value is `model_config.json`.
-    - `resource_files_names` (dict): use this to map resources to specific file
-      names for saving and loading.
-    - `pretrained_resource_files_map` (dict): The dict has the same keys as
-      `resource_files_names`, the values are also dict mapping specific pretrained
-      model name to URL linking to pretrained model.
-    - `pretrained_init_configuration` (dict): The dict has pretrained model names
-      as keys, and the values are also dict preserving corresponding configuration
-      for model initialization.
-    
-    - `base_model_prefix` (str): represents the the attribute associated to the
+    The base class for all pretrained models. It mainly provides common methods
+    for pretrained models loading (construction and loading) and saving. Loading
+    and saving also rely on the following class attributes which should be overridden
+    by derived classes accordingly:
+
+    - `model_config_file` (`str`): Represents the file name of model configuration
+      for configuration saving and loading in local file system. The value is
+      `model_config.json`.
+    - `resource_files_names` (`dict`): Represents resources to specific file
+      names mapping for resource saving and loading in local file system. Currently,
+      resources only include the model state, thus the dict only includes
+      `'model_state'` as key with corresponding value `'model_state.pdparams'`
+      for model weights saving and loading.
+    - `pretrained_init_configuration` (`dict`): Provides the model configurations
+      of built-in pretrained models (constract to models in local file system).
+      It has pretrained model names as keys (such as `bert-base-uncased`), and
+      the values are dict preserving corresponding configuration for model initialization.
+    - `pretrained_resource_files_map` (`dict`): Provides resource URLs of built-in
+      pretrained models (constract to models in local file system). It has the
+      same keys as `resource_files_names` (that is `'model_state'`), and the
+      values are also `dict` mapping specific pretrained model name (such as
+      `bert-base-uncased`) to URL of pretrained model weights.
```

Comment:
It has the same keys as `resource_files_names` (that is `'model_state'`), and the values are also `dict` mapping specific pretrained model name (such as `bert-base-uncased`) to URL of pretrained model weights.
->
It has the same key as `resource_files_names` (that is "model_state"), and the corresponding value is a dict with specific model name to model weights URL mapping (such as "bert-base-uncased" -> "https://paddlenlp.bj.bcebos.com/models/transformers/bert-base-uncased.pdparams").